src/main/kotlin/eu/europa/ec/eudi/verifier/endpoint/domain/Presentation.kt

```diff
@@ -167,6 +168,8 @@ sealed interface Presentation {
         val ephemeralEcPrivateKey: EphemeralEncryptionKeyPairJWK?,
         val responseMode: ResponseModeOption,
         val getWalletResponseMethod: GetWalletResponseMethod,
+        val keyMap: ConcurrentHashMap<String, ECPrivateKey>?,
```

Comment:
Personally I would prefer a naming that would not include the variable's type. `zkpKeys` or something should be enough I think.